{{cookiecutter.python_name}}/package.json

```diff
@@ -54,17 +55,25 @@
 {% endif %}
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
     "@jupyterlab/builder": "^3.1.0",
+    "@jupyterlab/galata": "^4.1.0",
```

Comment:
i dunno man... while webpack+babel and friends are heavy, the galata/playwright stack is _huge_, and i've seen a lot of experienced folk struggle with getting to _Works On My Machine_ with it. If this is going in-by-default, I feel like the pattern of putting the heavyweight browser tests in a separate `./ui-tests/package.json` and `yarn.lock` which can be fired off from the main `package.json` makes more sense.

From a CI value perspective: i've found it pays good dividends for heavyweight ui tests to be in an entirely separate, dependent, os-matrixed job where the "as-would-be-released" `.whl` and `.tar.gz` assets are installed and tested against. This allows the ui tests to fail _really fast_ if something isn't packaged properly, which is usually _much worse_ than being off by a few pixels in screenshot comparisons.